archive/templates/archive/objects/archive_filter.html

```diff
@@ -0,0 +1,187 @@
+{% load wagtailroutablepage_tags %}
+{% load archive_tags %}
+{% load humanize %}
+{% load static %}
+{% load wagtailcore_tags %}
+{% load wagtailimages_tags %}
+{% load wagtailuserbar %}
+{% load ubyssey_ad_tags %}
+
+{% if sections or years %}
+<div class="o-archive__sidebar show-desktop">
+    
+    <div class="o-archive__sidebar__filters">
+        <h2>Year</h2>
+        <ul>
+            <li
+                class="o-archive__sidebar__filter{% if not year %} o-archive__sidebar__filter--is-active{% endif %}">
+                <a href="?{% remove_field_from_query_string 'year' %}">All years</a>
+            </li>
+            {% for y in years %}
+            <li
+                class="o-archive__sidebar__filter{% if y == year %} o-archive__sidebar__filter--is-active{% endif %}">
+                <a href="?{% modify_query_string 'year' y %}">{{ y }}</a>
+            </li>
+            {% endfor %}
+        </ul>
+    </div>
+
+    <div class="o-archive__sidebar__filters">
+        <h2>Section</h2>
+        <ul>
+            <li
+                class="o-archive__sidebar__filter{% if not section_slug and video_section == False %} o-archive__sidebar__filter--is-active{% endif %}">
+                <a href="{% pageurl self %}{{ request|query_string }}">All sections</a>
+            </li>
+            {% for section in sections %}
+                {% if section.title != "The Ubyssey Student Marketplace" and "20" not in section.title and  section.title != "Pages" %}
```

Comment:
This seems like it will easily break. Perhaps instead we should add a checkbox field in sectionPage that, when checked, excludes the section from the Archive